NicolaeChristof/Rise
# Enhancement postprocessing

This PR changes a few aspects of our post-processing and general lighting. Most notably, it makes the game much brighter and the shadows less opaque. As for the post-processing, I changed the way to apply post-processing volumes so that we don't need an empty post-processing volume in the scene, instead just applying the volume to the cameras. I also changed the way that depth of field is controlled, so now we have one singular depth of field for each camera (instead of having to change each season's depth of field). Also, the depth of field changes as you zoom in and out.

How to Test:
Load up the different scenes and ensure that the post-processing is being applied to both cameras/all scenes.

Known Bugs:
Currently I'm having trouble with the tracking camera on the fall level not moving. If one of you could take a look at the fall scene when you test, that would be very helpful!
